core/templates/dev/head/pages/exploration_editor/settings_tab/settings_tab.html

```diff
@@ -586,3 +593,23 @@ <h3 ng-if="action === 'publicize_exploration'">Boost exploration ranking (as mod
     <button class="btn btn-danger" ng-click="reallyDelete()">Delete Exploration</button>
   </div>
 </script>
+
+<script type="text/ng-template" id="modals/previewSummaryTile">
+  <div class="modal-header">
+    <h3>A preview for your summary tile.</h3>
```

Comment:
I wonder if the user would know what a "summary tile" is.

How about "Preview Summary Card"?

That said -- I'm not sure they'll know what a 'summary card' is either. Do you think you could ask 2-3 friends (who don't know about Oppia) what they would call those "things" in the library page?